data/en/mbs_0204.json

```diff
@@ -0,0 +1,215 @@
+                            "answers": [{
+                                "id": "significant-change-established-answer",
+                                "label": "",
+                                "mandatory": true,
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ],
+                                "q_code": "146",
+                                "type": "Radio"
+                            }]
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "block": "change-comment-block",
+                                    "when": [{
+                                        "id": "significant-change-established-answer",
+                                        "condition": "equals",
+                                        "value": "Yes"
+                                    }]
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "group": "summary-group"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "type": "Question",
+                        "id": "change-comment-block",
+                        "questions": [{
+                            "answers": [{
+                                "id": "change-comment",
+                                "label": "Comments",
+                                "mandatory": true,
+                                "q_code": "146h",
+                                "type": "TextArea"
+                            }],
+                            "description": "We rely on your commentary to ‘tell the story’ behind changes in figures. By commenting here it will reduce the need for us to call you.",
+                            "id": "change-comment-question",
+                            "title": "Please describe the changes for {{respondent.trad_as_or_ru_name}} in more detail",
+                            "type": "General"
+                        }],
+                        "title": "Changes"
+                    }
+                ]
+            }]
+        },
+        {
+            "id": "summary-section",
+            "title": "Summary",
+            "groups": [{
+                "id": "summary-group",
+                "title": "Summary",
+                "blocks": [{
+                    "type": "Summary",
+                    "id": "summary"
+                }]
+            }]
+        }
+    ]
+}
```

Comment:
broken footer across form type

Reply:
I'm not seeing that. It'll be something to do with the NI theme not rendering correctly for you. We'll figure out what's going on locally there.